dataplane/Cargo.toml

```diff
@@ -6,11 +6,11 @@ publish = false
 license = "Apache-2.0"
 
 [dependencies]
-
 clap = { workspace = true, features = ["derive"] }
 ctrlc = { workspace = true, features = ["termination"] }
-net = { workspace = true, features = ["serde"] }
 dpdk = { workspace = true }
+iptrie = { workspace = true }
```

Comment:
Same here. Nothing wrong, you'll have to deal with it when merge/rebase.
Raising this because I'd actually merge https://github.com/githedgehog/dataplane/pull/211 first for the reason mentioned below.

Reply:
If it's the exact same change for this line, I expect git/GitHub is able to silently discard this chunk when rebasing, so it shouldn't create a merge conflict, whichever PR gets in first.